Modify installation instructions to use Mambaforge (#216)
astropy/astropy-workshop: 00-Install_and_Setup/README.md

To install WSL, you should follow the instructions Microsoft provides here: https://docs.microsoft.com/en-us/windows/wsl/install. While you may choose an alternative Linux distribution from the default Ubuntu, the instructions below have been tested on Ubuntu, so unless you have a specific reason, we suggest you stick with the default.  Once you reach the point in the instructions with a working Linux terminal prompt, you can proceed to step 1 of these instructions.

*Optional* While you can run a WSL terminal with the command prompt built into Windows, it's rather bare-bones and you may not have the best experience.  For WSL on Windows you'll probably want to [install Windows Terminal](https://docs.microsoft.com/en-us/windows/terminal/install) to have a terminal experience closer to what you'd see on Mac or Linux.

## 1. Install Mambaforge (if needed)

*Mambaforge is a free minimal installer for mamba. It is a small, bootstrap

> version of Anaconda that includes only mamba, Python, the packages they depend

Reviewer: ```suggestion
version of mamba that includes only mamba, Python, and a few packages they depend
```

on, and a small number of other useful packages, including pip, zlib and a few
others. Note, though, that if you have Mambaforge, Miniconda or the full Anaconda
already installed, you can skip to the next step.*

In a terminal window, check if Mambaforge is already installed.

    % mamba info

If Mambaforge is not already installed, obtain an installer for your
operating system: https://github.com/conda-forge/miniforge#mambaforge.
Then follow the installation instructions at

> https://mamba.readthedocs.io/en/latest/installation.html

Reviewer: ```suggestion
https://github.com/conda-forge/miniforge#mambaforge
```
The link you gave is useful for fine control over the installation, but the actual download and most-likely most-useful instructions are at the link I've suggested.
